app/assets/stylesheets/dashboards.scss

```diff
@@ -24,3 +24,28 @@
 .bootstrap_toggle_spacing .checkbox {
   margin-top: 0px;
 }
+
+.pt-phone,.pt-LMP {
+  width: 100px;
+  white-space: nowrap;
+  padding-right: 5px;
+}
+
+.pt-recent-notes {
```

Comment:
The note cell feels really cramped to me. Do you think it's a good idea to just reduce the size of the partial note we're displaying instead of splitting it onto two lines? Right now we're displaying the first 40 characters of a note and then the plus sign glyphicon, but I wonder if we want to decrease that so this fits better. (It would also let us ditch the glyphicon-plus-sign float below.) Thoughts?